.github/workflows/update.yml

```diff
@@ -41,12 +44,16 @@ jobs:
 
       # if event is coming from octokit/types.ts, update @octokit/types to latest version and get OpenAPI version from its package.json
       - run: npm install @octokit/types@latest
-        if: github.event_name == 'repository_dispatch' && github.event.action == 'octokit/types.ts release'
-      - run: node -e "console.log('::set-output name=version::' + require('@octokit/types/package').octokit['openapi-version'])"
-        if: github.event_name == 'repository_dispatch' && github.event.action == 'octokit/types.ts release'
+        if: github.event_name == 'repository_dispatch' && github.event.action ==
+          'octokit/types.ts release'
+      - run: node -e "console.log('::set-output name=version::' +
+          require('@octokit/types/package').octokit['openapi-version'])"
+        if: github.event_name == 'repository_dispatch' && github.event.action ==
+          'octokit/types.ts release'
         id: openapi_types
       - run: npm run update-endpoints
-        if: github.event_name == 'repository_dispatch' && github.event.action == 'octokit/types.ts release'
+        if: github.event_name == 'repository_dispatch' && github.event.action ==
+          'octokit/types.ts release'
```

Comment:
@oscard0m can you revert the style changes? Odd that this happened, don't you use prettier with default settings for formatting?

Reply:
Yep, I will revert them

---

I think I'm running prettier with no options so the default ones are applied but I will check what's happening.

Reply:
Found the issue, latest changes were not released in my octoherd script. I just run it in my fork and it worked: https://github.com/oscard0m/plugin-enterprise-server.js/pull/2

Reply:
Fixed in https://github.com/octokit/plugin-enterprise-server.js/pull/408/files